src/styles/index.scss

```diff
@@ -1,3 +1,117 @@
+@import './utils/variable';
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+/* #region card */
+
+.block {
+  box-sizing: border-box;
+  display: flex;
+  width: 200px;
+  background-color: #fff;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.image {
+  display: flex;
+  background-image: url(/src/images/imac.jpeg);
```

Comment:
The URL for the background image seems incorrect. Ensure the path is correct relative to the CSS file location. It should be './images/imac.jpeg' if the image is in the 'images' folder within the 'src' directory.